cmd/ocm-backplane/config/set.go

```diff
@@ -83,14 +87,17 @@ func setConfig(cmd *cobra.Command, args []string) error {
 		bpConfig.ProxyURL = &args[1]
 	case SessionConfigVar:
 		bpConfig.SessionDirectory = args[1]
+	case PagerDutyAPIConfigVar:
+		bpConfig.PagerDutyAPIKey = args[1]
 	default:
-		return fmt.Errorf("supported config variables are %s, %s & %s", URLConfigVar, ProxyURLConfigVar, SessionConfigVar)
+		return fmt.Errorf("supported config variables are %s, %s, %s & %s", URLConfigVar, ProxyURLConfigVar, SessionConfigVar, PagerDutyAPIConfigVar)
 	}
 
 	viper.SetConfigType("json")
 	viper.Set(URLConfigVar, bpConfig.URL)
-	viper.Set(ProxyURLConfigVar, bpConfig.ProxyURL)
+	viper.Set(ProxyURLConfigVar, &bpConfig.ProxyURL)
```

Comment:
any reason to set it via  `&bpConfig` pointer?

Reply:
That was unintended change. Thanks for catching it.